### PR TITLE
Added Notification Channels

### PR DESCRIPTION
**Pull request by MohitM90**
Notification Channels are required in Android 8+ for notifications to show up.

### CODE REVIEW

File: app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java [hunk 1]
```diff
@@ -37,6 +42,18 @@ public NotificationUtil(@ForApplication Context appContext) {
         this.appContext = appContext;
     }
 
+    public void createNotificationChannel() {
+        // Create the Notification Channel, but only on API 26+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            CharSequence name = appContext.getString(R.string.notification_text);
```
Comment (helloworld1) [inline app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java]: I would suggest putting a more sensible name for the channel name like "Review reminder", the name will show up in settings for people to toggle.

File: app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java [hunk 2]
```diff
@@ -27,6 +29,9 @@
 @PerApplication
 public class NotificationUtil {
     private static final String TAG = NotificationUtil.class.getSimpleName();
+
+    private final String CHANNEL_ID = "NOTIFICATION_CHANNEL";
```
Comment (helloworld1) [inline app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java]: Channel ID would be REVIEW_REMINDER

File: app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java (hunk 1 above)
Comment (MohitM90) [reply app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java]: Good point. I had only checked the German string for R.string.notification_text ("Erinnerungen"), which can be translated to "Reminders". I didn't notice that the English string is actually "Notification".

I've added better names as you suggested.

File: app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java (hunk 2 above)
Comment (MohitM90) [reply app/src/main/java/org/liberty/android/fantastischmemo/utils/NotificationUtil.java]: fixed :)

**Comment by helloworld1** [comment]
Thanks for making the change!

**Pull request merged**